controllers/admin.js

```diff
@@ -33,9 +33,12 @@ exports.postAddProduct = (req, res, next) => {
 };
 
 exports.getProducts = (req, res, next) => {
-  Product.find({ userId: req.user._id })
-    // .select("title price")
-    // .populate("userId", 'name')
+  Product.find()
+    // { userId: req.user._id }
+    .select("title price description image")
+    .populate("userId", "email")
+    //I changed the way with which we edit product since there will be only one admin,
+    //  and users cannot add or edit products at all.
```

Comment:
I changed the way with which we edit product since there will be only one admin,
  and users cannot add or edit products at all.